demo/index.html

```diff
@@ -99,6 +99,7 @@
 							<a href="/about">about</a>
 							<a href="/contact">contact</a>
 							<a href="/not-exist">not exist</a>
+							<a href="/lazy">lazy</a>
```

Comment:
I think we coult put directly the `/docs` path and if you do please move it one place up